src/pages/Books.jsx

```diff
@@ -5,11 +5,9 @@ import './page.scss';
 
 const BooksPage = () => (
```

Comment:
- [ ] Your app is missing some important sections right now which makes your stylings incomplete. Kindly make sure that your stylings are EXACTLY the same as the one provided in Zeplin. Font size, font weight positioning of elements, and layouts should be EXACTLY the same as the original design.

|  Zeplin  |  Yours  |
| --- | --- |
|  ![zeplin](https://user-images.githubusercontent.com/63087491/191862800-7515c4e1-0c02-498d-b60c-ffdbadb8ed6e.png)  |  ![annotely_image (11)](https://user-images.githubusercontent.com/63087491/191862664-81f97372-297b-4590-90a6-541e1aa8fbbb.png)  |